app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -653,17 +666,7 @@ public String getFragmentTag() {
 
     @Override
     public boolean onBackPressed() {
-        final boolean preview = (
-                getActivity().getIntent().getBooleanExtra(DocumentActivity.EXTRA_DO_PREVIEW, false)
-                        || _appSettings.getDocumentPreviewState(getPath())
-                        || _document.getFile().getName().startsWith("index."));
-        if (_isPreviewVisible && !preview) {
-            setDocumentViewVisibility(false);
-            return true;
-        } else if (!_isPreviewVisible && preview) {
-            setDocumentViewVisibility(true);
-            return true;
-        } else if (_menuSearchViewForViewMode != null && !_menuSearchViewForViewMode.isIconified()) {
```

Comment:
Now that home button is gone, back button directly exits even when in preview mode

Did this so that a user can directly exit from preview state. Without this change user is forced to go through edit more making remembering preview state useless.